coralproject/talk
# Akismet

## What does this PR do?

- Introduces the Akismet plugin which can send comments off to Akismet to flag possible spam comments as such

## How do I test this PR?

1. Create a comment with text "asdfasdf"
2. Notice the spam message: "The language in this comment looks like spam. You can edit the comment or submit it anyway for moderator review."
3. Submit the comment again and see: "Thank you. Our moderation team will review your comment shortly."
4. See the comment in the moderation panel with spam markers
